Tests/Marketplace/search_and_install_packs.py

```diff
@@ -134,45 +134,31 @@ def get_pack_dependencies(client: demisto_client, pack_data: dict, lock: Lock):
     pack_id = pack_data['id']
     logging.debug(f'Getting dependencies for pack {pack_id}')
     try:
-        try:
-            response_data, status_code, _ = demisto_client.generic_request_func(
-                client,
-                path='/contentpacks/marketplace/search/dependencies',
-                method='POST',
-                body=[pack_data],
-                accept='application/json',
-                _request_timeout=None,
-                response_type='object'
-            )
-        except ApiException as ex:
-            try:
-                logging.exception(f'Exception trying to get pack {pack_id} dependencies.'
-                                  f' Exception: {ex.status}, {ex.body}')
-            except Exception:
-                logging.error(f'An error occurred while parsing the dependencies result.'
-                              f' Rrror: {str(ex)}')
-                raise ex
-        except Exception as ex:
-            logging.exception(f'Exception trying to get pack {pack_id} dependencies.'
-                              f'  Exception: {ex}')
-
+        response_data, status_code, _ = demisto_client.generic_request_func(
+            client,
+            path='/contentpacks/marketplace/search/dependencies',
+            method='POST',
+            body=[pack_data],
+            accept='application/json',
+            _request_timeout=None,
+            response_type='object'
+        )
         if 200 <= status_code < 300:
             dependencies_data: list = []
             dependants_ids = [pack_id]
             response_data = response_data.get('dependencies', [])
             create_dependencies_data_structure(response_data, dependants_ids, dependencies_data, dependants_ids)
-            dependencies_str = ', '.join([dep['id'] for dep in dependencies_data])
             if dependencies_data:
+                dependencies_str = ', '.join([dep['id'] for dep in dependencies_data])
                 logging.debug(f'Found the following dependencies for pack {pack_id}: {dependencies_str}')
             return dependencies_data
         if status_code == 400:
             logging.error(f'Unable to find dependencies for {pack_id}.')
             return []
-        else:
-            msg = response_data.get('message', '')
-            raise Exception(f'Failed to get pack {pack_id} dependencies - with status code {status_code}\n{msg}\n')
-    except Exception:
-        logging.exception(f'The request to get pack {pack_id} dependencies has failed. {status_code=}.')
```

Comment:
Add `msg` in the log as well.

Reply:
there is no need, it will be added as the `{ex}` in the f-string, when the exception is caught.